tailwind.config.js

```diff
@@ -12,6 +27,7 @@ module.exports = {
       admissions: "url('/assets/backgrounds/admissions.jpg')",
       events: "url('/assets/backgrounds/events.jpg')",
       contact: "url('/assets/backgrounds/contact.jpg')",
+      aksaraBox: "url('/assets/backgrounds/aksaraBox.png')",
```

Comment:
Instead of using preloaded assets, can you change the `AksaraBox` component to render images from URLs? Cause each article will have its own image sent from the DB